Net22Solution/Net22Task/XMLToObjectDeserializer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+
+namespace Net22Task
+{
+    public class XMLToObjectDeserializer
```

Comment:
This class is too generic. 
Rename to ConfigXmlReader, drop Seralize function as per requirements the app only reads XML